homeassistant/components/music_assistant/strings.json

```diff
@@ -0,0 +1,57 @@
+{
+  "issues": {
+    "invalid_server_version": {
+      "title": "The Music Assistant server is not the correct version",
+      "description": "Check if there are updates available for the Music Assistant Server and/or integration."
+    }
+  },
+  "config": {
+    "step": {
+      "init": {
+        "data": {
+          "url": "URL of the Music Assistant server"
+        }
+      },
+      "manual": {
+        "title": "Manually add Music Assistant Server",
+        "description": "Enter the URL to your already running Music Assistant Server. If you do not have the Music Assistant Server running, you should install it first.",
```

Comment:
I kinda feel like this is always the case, as in, if I want to connect to a service I need to install it

Reply:
Yeah, well you would think so but in the end we did have to explicitly add this text because people do not understand it, I think its because MA is an addon included by default so you may miss the fact that you actually need to install that addon to get going. Lets please leave this text as we deliberatly added it after feedback